app/src/main/java/ch/epfl/cs311/wanderwave/ui/components/login/WelcomeTitle.kt

```diff
@@ -8,27 +8,32 @@ import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.platform.testTag
-import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.style.TextAlign
-import ch.epfl.cs311.wanderwave.R
 import ch.epfl.cs311.wanderwave.ui.components.animated.AnimatedAppIcon
 
 @Composable
-fun WelcomeTitle(modifier: Modifier) {
+fun AppLogoWithTitle(
+    modifier: Modifier,
+    initialColor: Color,
```

Comment:
Same here, the colors should be in the ressource folder (in the theme), you can very easily add material colors. Especially as we need to have dark and light theme at the end I think. It will be much easier like that.